x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/build_search_ui_config.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Schema } from '../../../../shared/types';
+
+export const buildSearchUIConfig = (apiConnector: object, schema: Schema) => {
+  return {
+    alwaysSearchOnInitialLoad: true,
+    apiConnector,
+    trackUrlState: false,
+    initialState: {
+      sortDirection: 'desc',
+      sortField: 'id',
+    },
+    searchQuery: {
+      result_fields: Object.keys(schema || {}).reduce(
```

Comment:
Ah it looks like the `|| {}` got added back in somehow by a newer commit

```suggestion
      result_fields: Object.keys(schema).reduce(
```